Faker.py

```diff
@@ -1,7 +1,29 @@
 #----------------------------------------------------------------------------
 # Created By  : Henry Dalton
-# Created Date: 29/12/2021 
+# Created Date: 29/12/2021
 # version ='1.0'
 # ---------------------------------------------------------------------------
-""" """  
+
 # ---------------------------------------------------------------------------
+
+import pandas as pd
+
+def Faker_creator(num_people, list_fields, cust_dict):
```

Comment:
Use more descriptive names. e.g. `def fake_people_dataframe(num_people, standard_fields, custom_dict)`
Long variable names are fine, the more your code reads like english the better